leaf-ui/js/backendComm.js

```diff
@@ -142,18 +142,21 @@ function errorExists(analysisResults) {
  * @returns {boolean}
  */
 function getErrorMessage(backendErrorMsg) {
-
 	// If node ids does not exist, just return the original error message for now
-	if (!nodeIDsExists(backendErrorMsg)) {
+	if (!nodeNumsExists(backendErrorMsg)) {
 		return backendErrorMsg;
 	}
 
-	var ids = getIDs(backendErrorMsg);
+	var ids = getNums(backendErrorMsg);
```

Comment:
perhaps this variable could be `nums` rather than `ids`, now that we're calling them nums everywhere else?